code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubConfigManager.java

```diff
@@ -35,6 +35,7 @@
     private static final String CLIENT_SECRET = "clientSecret";
     private static final String ALLOWED_USERS = "allowedUsers";
     private static final String ALLOWED_ORGS = "allowedOrganizations";
+    private static final String HOST_NAME = "hostName";
```

Comment:
Can you make hostname lowercase.  I know host and name are two different words but geeks have unanimously agreed that hostname should be a word